src/train_model.py

```diff
@@ -467,6 +535,10 @@ def train_and_save(network, optimizer, criterion, train_loader,
                              cluster_gradient is True.
     decay_lr: bool representing whether or not to decay the learning rate over
               training.
+    calc_simple_math_diags: bool for whether to calculate the additional diags
```

Comment:
oh - does 'diags' mean 'diagnostics'? That's not how I read it, plz spell the word out in this comment.